DesktopModules/Vanjaro/UXManager/Library/Base.ascx.cs

```diff
@@ -144,8 +145,9 @@ protected void Page_Load(object sender, EventArgs e)
 
         protected void Page_PreRender(object sender, EventArgs e)
         {
+
             if (InjectEditor())
-                WebForms.RegisterClientScriptBlock(Page, "EditorInit", "$(document).ready(function(){ if(typeof GrapesjsInit !='undefined') GrapesjsInit(" + JsonConvert.SerializeObject(Editor.Settings) + "); });", true);
+                WebForms.RegisterClientScriptBlock(Page, "EditorInit", "var TemplateLibraryURL = \"" + TemplateLibraryURL + "\"; var ThemeGUID = \"49A70BA1-206B-471F-800A-679799FF09DF\"; $(document).ready(function(){ if(typeof GrapesjsInit !='undefined') GrapesjsInit(" + JsonConvert.SerializeObject(Editor.Settings) + "); });", true);
```

Comment:
Why is the theme GUID hardcoded? Be sure to use the ThemeManager.CurrentTheme.Guid which Naren is working on in #149